### PR TITLE
[FormRecognizer] Remove <inheritdoc> from documentation

### PR DESCRIPTION
**Pull request by kinelski**
Removing all `<inheritdoc>` occurrences from the Form Recognizer SDK. Due to a known bug (#8503), github.io docs for a class member are not populated if the `<inheritdoc>` tag is used.

### CODE REVIEW

File: sdk/core/Azure.Core/src/Operation{T}.cs [hunk 1]
```diff
@@ -53,8 +53,8 @@ public abstract class Operation<T> where T : notnull
         /// <summary>
         /// Periodically calls the server till the long-running operation completes.
         /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> used for the periodical service calls.</param>
+        /// <returns>The last HTTP response received from the server.</returns>
```
Comment (kinelski) [inline sdk/core/Azure.Core/src/Operation{T}.cs:57]: In parallel to the Form Recognizer changes, adding some missing info in `Operation{T}`.

**Comment by kinelski** [comment]
@pakrym Found this in `HttpMessage` (it seems a copy-paste mistake):
https://github.com/Azure/azure-sdk-for-net/blob/2e9a93c8ea2e6f71c7f898455167a2c8ad912880/sdk/core/Azure.Core/src/HttpMessage.cs#L62-L65

I can update it in this PR, but I don't really know what this `CancellationToken` does here. Any suggestions?

**Comment by pakrym** [comment]
> I can update it in this PR, but I don't really know what this CancellationToken does here. Any suggestions?

The CancellationToken to be used during the HttpMessage processing

**Review by pakrym — APPROVED** [review]
Core docs LGTM

**Review by maririos — APPROVED** [review]

**Pull request merged**